src/sentry/static/sentry/app/components/group/releaseChart.jsx

```diff
@@ -21,15 +18,16 @@ const GroupReleaseChart = createReactClass({
     firstSeen: PropTypes.string,
     lastSeen: PropTypes.string,
     title: PropTypes.string,
-  },
+  };
 
-  getInitialState() {
-    return this.getNextState(this.props);
-  },
+  constructor(props) {
```

Comment:
I think minorly more correct would be `...args` and `super(...args)`.

Reply:
React docs always use `props` fwiw